skyreader/plot/plot.py

```diff
@@ -386,138 +489,280 @@ def bounding_box(ra, dec, theta, phi):
                     tmp_dec = np.pi/2 - tmp_theta
                     tmp_ra = tmp_phi
                     grid_map[(tmp_dec, tmp_ra)] = value
+                    # nested_pixel = healpy.ang2pix(
+                    #     nside, tmp_theta, tmp_phi, nest=True
+                    # )
+                    # uniq = 4*nside*nside + nested_pixel
+                    # uniq_list.append(uniq)
             LOGGER.info(f"done with map for nside {nside}...")
 
         grid_dec_list, grid_ra_list, grid_value_list = [], [], []
 
         for (dec, ra), value in grid_map.items():
-            grid_dec_list.append(dec); grid_ra_list.append(ra)
+            grid_dec_list.append(dec)
+            grid_ra_list.append(ra)
             grid_value_list.append(value)
         grid_dec: np.ndarray = np.asarray(grid_dec_list)
         grid_ra: np.ndarray = np.asarray(grid_ra_list)
         grid_value: np.ndarray = np.asarray(grid_value_list)
+        # uniq_array: np.ndarray = np.asarray(uniq_list)
 
         sorting_indices = np.argsort(grid_value)
         grid_value = grid_value[sorting_indices]
         grid_dec = grid_dec[sorting_indices]
         grid_ra = grid_ra[sorting_indices]
+        # uniq_array = uniq_array[sorting_indices]
 
         min_value = grid_value[0]
         min_dec = grid_dec[0]
         min_ra = grid_ra[0]
 
-        LOGGER.info(f"min  RA: {min_ra *180./np.pi} deg, {min_ra*12./np.pi} hours.")
+        LOGGER.info(
+            f"min  RA: {min_ra * 180./np.pi} deg, {min_ra*12./np.pi} hours."
+        )
         LOGGER.info(f"min Dec: {min_dec * 180./np.pi} deg")
 
         # renormalize
         grid_value = grid_value - min_value
         min_value = 0.
 
-        # show 2 * delta_LLH
-        grid_value = grid_value * 2.
-
-        # Do same for the healpy map
+        # renormalize
         equatorial_map[np.isinf(equatorial_map)] = np.nan
         equatorial_map -= np.nanmin(equatorial_map)
-        equatorial_map *= 2.
 
-        LOGGER.info(f"preparing plot: {plot_filename}...")
+        if llh_map:
+            # show 2 * delta_LLH
+            grid_value = grid_value * 2.
+            equatorial_map *= 2.
+        else:
+            # Convert to probability
+            equatorial_map = np.exp(-1. * equatorial_map)
+            equatorial_map = equatorial_map / np.nansum(equatorial_map)
+
+            # nan values are a problem for the convolution and the contours
+            min_map = np.nanmin(equatorial_map)
+            equatorial_map[np.isnan(equatorial_map)] = min_map
+
+            if angular_error_floor:
+                # convolute with a gaussian with 0.2 deg as sigma
+                equatorial_map = healpy.smoothing(
+                    equatorial_map,
+                    sigma=np.deg2rad(self.NEUTRINOFLOOR_SIGMA),
```

Comment:
The idea was to apply exactly what is done offline, and offline 0.2 deg is used as the sigma (so with 39% coverage). I know that for the first GCN Notice we use 0.2 deg for the 50% error instead, but since also there the plan is to apply exactly the same as offline, I guess this would be the solution which makes most sense.